### ISSUE TITLE
WavPack export: include MD5 audio checksum (-m option)

### ISSUE THREAD

**Issue by YZgitter** [audacity/audacity#5635]
### Your idea

WavPack files can store a MD5 checksum of the audio (just like FLAC). I think Audacity should do this by default.

### Problem to be solved

Integrity verification is important, especially for a format like WavPack, which is often used for lossless archival.

**Comment by LWinterberg** [comment]
are there any drawbacks whatsoever to storing that bit of metadata? If all it does is add like 60 bytes worth of metadata to the file which can safely be ignored if you don't need it, it indeed would make sense to just plop it in

**Comment by YZgitter** [comment]
Encoding is probably a bit slower because of the calculation, but I really doubt that it's a significant, perceptible difference (I didn't do any measurements, however).

**Comment by dg-no-9** [comment]
Can I work on this item since it is labeled as 'good first issue'?

**Comment by LWinterberg** [comment]
@dg-no-9 absolutely. And if anyone else comes across this: Just submit a PR if you've figured it out :)

**Comment by ajsand** [comment]
Hey, I am working on this now. I'll submit a PR if I manage to figure it out.

**Comment by kryksyh** [comment]
@ajsand awesome! Make sure you branch off `audacity3` branch.

**Comment by chinakov** [comment]
Tested, wavpack files now contain MD5 hash on export. Checked via wvunpack CMD tool. 

<img width="1113" height="759" alt="Image" src="https://github.com/user-attachments/assets/4285b23b-08ed-42c2-ab97-a95c20543775" />

First file is exported on new PR, second file is from Release.
